packages/test/test-gc-sweep-tests/package.json

```diff
@@ -12,6 +12,7 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
+	"type": "module",
```

Comment:
it's not clear to me how this package's scripts work or why it doesn't cause ci to fail: `npm run test` fails locally, but if I use what looks like the intended run target, the test passes. It does seem like the tests are not supposed to run in CI based on the package readme and script doc, but figured I'd call this out.

Reply:
These tests are not used. We can delete this package.

Reply:
+ @tyler-cai-microsoft In case there is something that should be preserved.